docs/cheatsheet.rst

```diff
@@ -0,0 +1,111 @@
+Cheatsheet
+----------
```

Comment:
This header `---...` needs to be changed to something higher level (or the ones below to lower level). Double check with what's used for rst and across the cartoframes docs.